restclients_core/dao.py

```diff
@@ -221,6 +222,14 @@ def get_setting(self, key, default=None):
         key = "RESTCLIENTS_%s" % key
         return getattr(settings, key, default)
 
+    def get_overridable_setting(self, key, default):
```

Comment:
There is already a get_service_setting

Reply:
I may be mistaken, but I believe that get_service_setting does not allow to inherit the default global restclient setting.

Reply:
How about add default to get_service_setting?